packages/firestore/src/local/local_store.ts

```diff
@@ -1049,11 +1114,13 @@ export class MultiTabLocalStore extends LocalStore {
   }
 
   setNetworkEnabled(networkEnabled: boolean): void {
-    this.persistence.setNetworkEnabled(networkEnabled);
+    (this.persistence as IndexedDbPersistence).setNetworkEnabled(
```

Comment:
If you add a `persistence: IndexedDbPersistence` field and assign it manually (similar to mutationQueue), this cast will go away.

Reply:
Done.